src/appleseed/renderer/modeling/project/projectfileupdater.cpp

```diff
@@ -2017,6 +2018,55 @@ namespace
             }
         }
     };
+
+
+    //
+    // Update from revision 30 to revision 31.
+    //
+
+    class UpdateFromRevision_30
+      : public Updater
+    {
+      public:
+        explicit UpdateFromRevision_30(Project& project)
+          : Updater(project, 30)
+        {
+        }
+
+        void update() override
+        {
+            replace_max_samples_interactive_renderer_setting();
+        }
+
+      private:
+        void replace_max_samples_interactive_renderer_setting()
+        {
+            for (Configuration& config : m_project.configurations())
+            {
+                Dictionary& root = config.get_parameters();
+
+                if (root.dictionaries().exist("progressive_frame_renderer")) {
+                    Dictionary& pfr = root.dictionaries().get("progressive_frame_renderer");
+
+                    if (pfr.strings().exist("max_samples")) {
+                        RENDERER_LOG_WARNING(
```

Comment:
I don't think we need this warning since we can migrate the project in a (mostly, admittedly) compatible manner.

The previous upgrade emitted a warning because there was no easy way to migrate the Adaptive Pixel Renderer settings to the new Adaptive Tile Renderer.